tests/AArch64/Run.cpp

```diff
@@ -300,6 +300,7 @@ static void RunWithFlags(const test::TestInfo *info,
   // native program state recorded before executing the native testcase,
   // but after swapping execution to operate on `gStack`.
   if (!sigsetjmp(gJmpBuf, true)) {
+    std::fesetenv(FE_DFL_ENV);
```

Comment:
Does this require `#pragma STDC FENV_ACCESS ON`?

Do we want to warn if `std::fesetenv()` returns a non-zero value, as in it failed to control the FP environment?

Reply:
Clang doesn't support the pragma, and libc correctly implements the function. It's more that LLVM doesn't yet understand that these functions mutate the floating point environment, and therefore affect optimizations of nearby floating point code.